test/e2e/common.go

```diff
@@ -20,6 +20,19 @@ func newPod(namespace string, name string, containerName string, runtimeclass st
 	}
 }
 
+// newPod returns a new Pod object.
+func newBusyboxPod(namespace string, name string, containerName string, runtimeclass string) *corev1.Pod {
+	return &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: namespace},
+		Spec: corev1.PodSpec{
+			Containers:       []corev1.Container{{Name: containerName, Image: "quay.io/prometheus/busybox:latest", Command: []string{"/bin/sh", "-c", "sleep 3600"}}},
```

Comment:
Does `quay.io/prometheus/busybox:latest` support both `amd64` and `s390x`?

Reply:
yes it supports multiple architectures including `amd64` and `s390x`

```
sudharshanmuralidharan@Sudharshans-MacBook-Pro ~ % docker manifest inspect quay.io/prometheus/busybox:latest
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:63f94053d90ec5e67a7ea233a60163e48a1f36ca7fd423bed473d6deb5805860",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:980ce93918f50624dec1f7a011f5c62ff58c122c3616959f149c612f5f24776a",
         "platform": {
            "architecture": "arm64",
            "os": "linux",
            "variant": "v8"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 737,
         "digest": "sha256:715a6e0fb54b473e5f0c1c41c199a8819f9764252e2fb1cde58caa6e17cd5443",
         "platform": {
            "architecture": "arm",
            "os": "linux",
            "variant": "v7"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 738,
         "digest": "sha256:b6f92704c8cf43a643f3b66b6a745996985c958afa8d35bca8c3078044c487d7",
         "platform": {
            "architecture": "ppc64le",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 738,
         "digest": "sha256:dc8003e0042ac173a22497ccd341cee636b89c2367b33d08c06726b54b5e3b3e",
         "platform": {
            "architecture": "s390x",
            "os": "linux"
         }
      }
   ]
}

```